Add post: Emailing users who try to sign in with an unregistered email (#249)
DFE-Digital/teacher-cpd-design-history: app/posts/manage-training/2023-09-14-emailing-users-who-try-to-sign-in-with-an-unregistered-email.md

---
title: Emailing users who try to sign in with an unregistered email
description: We started emailing users who tried to sign in with an unregistered email address,
date: 2023-09-14
tags:
  - authentication
  - accounts
image:
  src: /manage-training/emailing-users-who-try-to-sign-in-with-an-unregistered-email/access-the-service.png
  alt: "Illustration showing an email saying ‘Someone tried to access the service with this email address. This email address is not registered with our service.’"
  opengraphImage: true
---
{% from "email/macro.njk" import appEmail %}

> We now send an email to anyone who tries to sign in to the service with an unregistered email address, to tell them how they can access the service.

Reviewer: ```suggestion
We now send an email to anyone who tries to sign in to the service with an unregistered email address. We provide guidance on how they can access the service.
```

## The issue

> The first page of the service invites people to either sign in, or else to request access to the service. Users who request access to the service can then receive a link sent to their school email address which will give them access.

Reviewer: ```suggestion
The first page of the service invites people to either sign in, or request access to the service. Users who request access to the service can generate an access link for their school. This is sent to the email address held on GIAS.
```
